pkg/controller/replicaset/replica_set_test.go

```diff
@@ -1738,3 +1740,17 @@ func TestRemoveCondition(t *testing.T) {
 		}
 	}
 }
+
+func resourcePath(resource, namespace, name string) string {
+	path := fmt.Sprintf("/apis/%s/%s/", apps.SchemeGroupVersion.Group, apps.SchemeGroupVersion.Version)
```

Comment:
Since this PR tries to switch RS controller to use apps/v1beta2 APIs, the controller itself wouldn't work without apps/v1beta2 being enabled